proto/sdk/sdk.proto

```diff
@@ -5,8 +5,9 @@ package sdk;
 option go_package = "github.com/ava-labs/avalanchego/proto/pb/sdk";
 
 message PullGossipRequest {
-  bytes filter = 1;
+  reserved 1;
```

Comment:
Should we comment on why it's reserved? Also add a TODO on removing this reserved field? post-v1.11?